javascript/javascript1/week3/q4.js

```diff
@@ -0,0 +1,30 @@
+const notes = [];
+
+    function saveNote(content, id) {
+        var arr=[];
```

Comment:
instead of defining another array(array of arrays is not ideal but)you can use object notation to push the notes into an array(array of objects). like this
 function saveNote(content, id) {
  notes.push({
      content ,
      id,
  })
}